.github/workflows/nextjs.yml

```
@@ -1,96 +1,84 @@
           node-version: "20"
           cache: ${{ steps.detect-package-manager.outputs.manager }}
-          cache-dependency-path: ${{ env.WORKSPACE_PATH }}/${{ steps.detect-package-manager.outputs.manager == 'yarn' && 'yarn.lock' || 'package-lock.json' }}
-      - name: 🔧 Setup Pages
+          cache-dependency-path: roamance/${{ steps.detect-package-manager.outputs.manager == 'yarn' && 'yarn.lock' || 'package-lock.json' }}
+
+      - name: ⚙️ Setup Pages
         uses: actions/configure-pages@v5
         with:
-          working-directory: ${{ env.WORKSPACE_PATH }}
-          static_site_generator: next
+          working-directory: roamance
```

actionlint (1.7.4)

input "working-directory" is not defined in action "actions/configure-pages@v5". available inputs are "enablement", "generator_config_file", "static_site_generator", "token"

(action)